plugins/woocommerce/client/blocks/assets/js/base/components/cart-checkout/return-to-cart-button/index.tsx

```diff
@@ -12,24 +12,27 @@ import './style.scss';
 interface ReturnToCartButtonProps {
 	href?: string | undefined;
 	children: React.ReactNode;
+	element?: React.ElementType;
 }
 
 const ReturnToCartButton = ( {
 	href,
 	children,
+	element = 'a',
 }: ReturnToCartButtonProps ): JSX.Element | null => {
 	const cartLink = href || CART_URL;
 	if ( ! cartLink ) {
 		return null;
 	}
+	const Element = element;
 	return (
-		<a
+		<Element
 			href={ cartLink }
```

Comment:
We should probably not put href on it if it's a span
```suggestion
			{ ...( element === 'a' ? { href: cartLink } : {} ) }
```